src/ts/contentScripts/live-preview/index.tsx

```diff
@@ -0,0 +1,279 @@
+import {Feature, Settings} from '../../utils/settings'
+import {Navigation} from '../../roam/navigation'
+import {browser} from 'webextension-polyfill-ts'
+import {createPopper, Instance} from '@popperjs/core'
+
+export const config: Feature = {
+    id: 'live_preview',
+    name: 'Live Preview',
+    warning: 'Experimental feature; Large databases might see performance issues.',
+    enabledByDefault: false,
+}
+const checkSettingsAndSetupIframe = () => {
+    Settings.isActive('live_preview').then(active => {
+        setupIframe(active)
+    })
+}
+checkSettingsAndSetupIframe()
+
+browser.runtime.onMessage.addListener(async message => {
```

Comment:
👀

Reply:
@Stvad doing this causes the iframe removal to stop working, because of the same instance problem -- on toggle, it reloads this script, resulting in the instance being `null`. 

If used via onMessage, it works fine. 


Changes I made (which caused it to break
- in live-preview, remove this `browser.runtime.onMessage.addListener` handling. 

- in `dispatcher`

```
 ['settings-updated', () => {
        updateShortcuts()
        checkSettingsAndSetupIframeToggle()
    }
```

- now on toggle, it does not `.destroy()` as it does not find the instance.